api/src/main/java/org/openmrs/util/Format.java

```diff
@@ -42,7 +42,7 @@ public static String formatPercentage(Number pct) {
 	}
 	
 	public static String format(double d) {
-		return "" + ((d == (int) d) ? (int) d : d);
```

Comment:
Are you sure this does not change the return value?

Reply:
Yes, look. If d is integer type, then we return (int) d. Else we return just d. So it can be simplified to return d :) I can write tests so you can be sure.